steps/github-script-runner/0.0.1/assets/icon.svg

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
assets dir should go to the step group dir (`steps/github-script-runner/assets`), not under a specific version, example: https://github.com/bitrise-io/bitrise-steplib/tree/master/steps/activate-ssh-key